websites/B/Bungie/presence.ts

```diff
@@ -0,0 +1,303 @@
+//}
+//else if (path.includes ("/collections/community-artist-series")){
+ // presenceData.startTimestamp = browsingStamp;
+ // presenceData.details = ("Seeing the Community Artist Series on the Bungie Shop")
+//}
+//else if (path.includes ("/pages/community-artist-series-guide")){
+ // presenceData.startTimestamp = browsingStamp;
+ // presenceData.details = ("Seeing what Community Artist Series Guide on the Bungie Shop")
+//}
+//else if (path.includes ("/collections/bungie-rewards")){
+ // presenceData.startTimestamp = browsingStamp;
+ // presenceData.details = ("Seeing what Bungie Rewards are about on the Bungie Shop")
+//}
+//else if (path.includes ("/pages/bungie-rewards-guide")){
+ // presenceData.startTimestamp = browsingStamp;
+ // presenceData.details = ("Seeing what Bungie Rewards rewards are available on the Bungie Shop")
+//}
+//else if (path.includes ("/collections/last-chance")){
+ // presenceData.startTimestamp = browsingStamp;
+ // presenceData.details = ("This is your last chance to get these items on the Bungie Shop!")
+//}
+//else if  (path.includes ("/collections/whats-new/products/preorder")){
+  //console.log(titleTwentyOne)
+  //titleTwentyOne = document.querySelector("#product > div.row.clearfix > div:nth-child(3) > h1 > div")
+  //titleTwentyTwo = document.querySelector("#price-preview")
+  //presenceData.startTimestamp = browsingStamp;
+  //presenceData.details = ("Looking at "+titleTwentyOne.innerHTML.replace ('<div name="secomapp-fg-image-4456031551575" style="display: none;"> <img src="//cdn.shopify.com/s/files/1/0417/0233/t/21/assets/icon-freegift.png?v=10831135327727470632" alt="Free Gift" class="sca-fg-img-label"> </div>',"") + "Price: "+titleTwentyTwo.innerHTML)
+//}
+//else if  (path.includes ("/collections/whats-new/products/")){
+  //titleTwentyThree = document.querySelector("#product > div.row.clearfix > div:nth-child(3) > h1 > div")
+  //document.querySelector("#product > div.row.clearfix > div:nth-child(3) > h1 > div")
+  //document.querySelector("#product > div.row.clearfix > div:nth-child(3) > h1")
+  //<div style="position:relative;"> Fading Light T-Shirt <div name="secomapp-fg-image-4358729597015" style="display: none;"> <img src="//cdn.shopify.com/s/files/1/0417/0233/t/21/assets/icon-freegift.png?v=10831135327727470632" alt="Free Gift" class="sca-fg-img-label"> </div> </div>
+  //document.querySelector("#product > div.row.clearfix > div:nth-child(3) > h1 > div")
+  //titleTwentyFour = document.querySelector("#product > div.row.clearfix > div:nth-child(3) > div.purchase > h2")
+  //presenceData.startTimestamp = browsingStamp;
+  //presenceData.details = ("Looking at "+ titleTwentyThree.innerHTML.replace ('<div name="secomapp-fg-image-4358729597015" style="display: none;"> <img src="//cdn.shopify.com/s/files/1/0417/0233/t/21/assets/icon-freegift.png?v=10831135327727470632" alt="Free Gift" class="sca-fg-img-label"> </div>',"")+ " Price: "+titleTwentyFour.innerHTML)
+//}
```

Comment:
Why is this not enabled?

Reply:
That is because that is the store, and I thought it would be easy to implement, but it turns out I was wrong. So, I decided to not delete, but to comment it out so that it doesn't do anything for the time being.